File: benchmark/results/java-invocation-script/run-004/a-mcp/output/ordering/value_objects/OrderLine.java
package com.metaengine.demo.ordering.value_objects;

import java.time.OffsetDateTime;

// OrderLine value object.

public class OrderLine {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public OrderLine(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/ordering/value_objects/order_line.py
"""OrderLine value object for the ordering domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class OrderLine:
    """Immutable value object for a single line within an order."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
